swagger/public_v1/component_schemas/SubjectAttributes.yml

```diff
@@ -1,12 +1,22 @@
 ---
 description: "This schema is used to describe a subject."
 type: object
-required:
-  - code
 properties:
   name:
     type: string
     description: "Subject name."
   code:
     type: string
     description: "Unique subject code."
+  bursary_amount:
+    type: string
+    description: "non-repayable sum of money that universities can award to students to incentivise and support their studies"
+  early_career_payments:
+    type: string
+    description: "Eligible mathematics, physics, chemistry and languages teachers can apply for early-career payments of up to £9,000 after tax."
+  scholarship:
+    type: string
+    description: "a grant or payment made to support a student’s education, awarded on the basis of academic or other achievement."
+  subject_knowledge_enhancement_course_available:
+    type: boolean
+    description: "SKE courses are available in: maths, physics, languages, chemistry, computing, biology, geography, English, religious education, design and technology, and primary maths. Schools or universities will identify your need for an SKE as part of their selection process, usually at interview. If they feel you need to enhance your knowledge, but have potential to be a great teacher, they’ll offer you a teacher training place on the condition that you complete an SKE course."
```

Comment:
should we actually be stating the subjects
if it changes then this needs to change